packages/mirrorful/editor/src/blocks.css

```
@@ -1,4 +1,4 @@
-@import '../.mirrorful/theme.css';
+/* @import '../.mirrorful/theme.css'; */
```

```suggestion
@import '../.mirrorful/theme.css';
```

@Bismay5467 why was this removed?